src/services/dashboards/dashboards.svc.coffee

```diff
@@ -110,12 +112,20 @@ angular
               orgId = success.currentOrganization.id
 
               # Retrieve dashboards with widgets and kpis settings
-              dashboardsPromise = $http.get(ImpacRoutes.dashboards.index(orgId)).then(
-                (response) ->
-                  _self.setDashboards(response.data).then(-> _self.setCurrentDashboard() )
-                (error) ->
-                  $log.error("Impac! - DashboardsSvc: cannot retrieve dashboards list for org: #{orgId}")
-              )
+              dashboardsPromise = $http.get(ImpacRoutes.dashboards.index(orgId))
+                .then(
+                  (success) ->
+                    success.data.forEach (o) ->
+                      o.widgets.forEach (widget) ->
+                        ImpacDateFormatter.formatDateString widget, widget.endpoint
```

Comment:
This is too intrusive. See comment in `DatesFormatterSvc` => I think the formatting should be done in the views rather than directly on the object.